runtime/indent/testdir/html.ok

```diff
@@ -41,6 +41,11 @@ div#d2 { color: green; }
 		dt text
 	    </dt>
 	</dl>
+	<div
+		class="test"
+		style="color: yellow">
+	    text
+	</div>
```

Comment:
Note that with the combination of sw=4 and ts=8 (which is the default, I just added it above to make it explicit), this ends up being a mix of tabs and spaces.
Since Github displays tabs with a width of 4, this is not visible here.
If this file was changed to either sw=8 or ts=4, this would be properly visible here. I can change that if preferred, but I did not want to change the indentation of the entire file if not necessary.